hermeto/core/package_managers/pip.py

```diff
@@ -72,9 +72,7 @@
 ZIP_FILE_EXT = ".zip"
 COMPRESSED_TAR_EXT = ".tar.Z"
```

Comment:
nitpick: Pre-existing, but since you're touching the code you could just add one trivial change - move ^this one to the list of `SDIST_FILE_EXTENSIONS` and hard-code it on the single spot where it's used to skip metadata parsing. The reasoning behind this is that it's confusing because `.tar.gz, .tar.bz2, .tar.xz` are all compressed, it's just that `.tar.Z` is older and as a matter of fact `tarfile` doesn't support it, no idea why we're not attempting to read the metadata (historical reasons...)